src/components/Notifications/NoticeBoard.tsx

```
@@ -1,42 +1,117 @@
+  }) => {
+    const formattedMessage = formatMessage(message);
+
+    return (
+      <>
+        <h1 className="font-semibold text-lg text-black mb-1 truncate hover:text-clip">
+          {" "}
+          {message.split("\n")[0]}
+        </h1>
+        <p
+          className={classes}
+          dangerouslySetInnerHTML={{ __html: formattedMessage }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)